### ISSUE TITLE
Add define block

### ISSUE THREAD

**Issue by SebCanet** [BlocklyDuino/BlocklyDuino-v2#6]

**Comment by sam-tj** [comment]
would have to make a block just like for variables but should have properties of definition block.

**Comment by sam-tj** [comment]
Try define block on this https://sam-tj.github.io/Turtle-Studio-v1/

If works well will add this here too.

**Comment by SebCanet** [comment]
Seems to be good, but there's a problem when you comment the block, the comment in code appears in loop.

**Comment by sam-tj** [comment]
this 
![image](https://user-images.githubusercontent.com/20317637/92478990-96ac0880-f200-11ea-9eeb-153e688795de.png)

**Comment by SebCanet** [comment]
yes...helas

**Comment by sam-tj** [comment]
yeah,that will be a problem. The data for that is populating from contextual menu and by default that comment goes to loop section.

**Comment by sam-tj** [comment]
any idea about disabling Add comment option for some blocks?

**Comment by SebCanet** [comment]
I prefer understanding how comments work and hack this, I think this could be a good question on the Blockly's forum :-D

**Comment by sam-tj** [comment]
yes that would be great.
Till then I have disabled the context menu for this define block.
Using:  this.contextMenu=false;
